Gemfile

```diff
@@ -4,6 +4,7 @@ ruby '2.4.1'
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
 gem 'rails', '~> 5.2.2'
 
+gem 'active_storage_base64'
```

Comment:
We won't be uploading many new versions to it, but for consistency maybe we can specify the version here.